static-pages/service-unavailable/service-unavailable.html

```diff
@@ -0,0 +1,114 @@
+                        </svg>
+                        <!--<![endif]-->
+                        <!--[if IE 8]>
+                            <img src="./assets/images/govuk-logotype-tudor-crown.png" class="govuk-header__logotype-crown-fallback-image" width="32" height="30" alt="">
+                        <![endif]-->
+                        <span class="govuk-header__logotype-text">
+                            GOV.UK
+                        </span>
+                    </span>
+                </a>
+            </div>
+            <div class="govuk-header__content">
+                <a href="/" class="govuk-header__link govuk-header__service-name">
+                    Check if you are eligible for an energy grant for a home with no mains gas boiler
+                </a>
+            </div>
+        </div>
+    </header>
+    <div class="govuk-width-container">
+        <div class="govuk-phase-banner">
+            <p class="govuk-phase-banner__content">
+                <strong class="govuk-tag govuk-phase-banner__content__tag">Beta</strong>
+                <span class="govuk-phase-banner__text">This is a new service – your <a class="govuk-link" href="https://forms.office.com/e/tsn29X5m91" target="_blank">feedback (opens in new tab)</a> will help us to improve it.</span>
+            </p>
+        </div>
+        <nav>
+            <a href="https://www.gov.uk/apply-home-upgrade-grant" class="govuk-back-link" id="back-link">Back</a>
```

Comment:
this goes back to the landing page, which I think will have further info on it? can remove if not